src/components/ApplicationViews.js

```diff
@@ -1,46 +1,26 @@
-import React, { Component } from "react";
-import { Route, Redirect } from "react-router-dom"
-import Login from "./auth/Login";
-import CreateAccount from "./auth/CreateAccount";
-import Home from "./songs/SongCollection";
-import SongView from "./songs/Song";
+import React, { Fragment } from 'react'
+import { Route, Redirect } from 'react-router-dom'
+import Login from './auth/Login'
+import { CreateAccount } from './auth/CreateAccount'
+import Home from './songs/SongCollection'
+import SongView from './songs/Song'
 
-class ApplicationViews extends Component {
```

Comment:
When a component only returns JSX.... it should _always_ be a functional component. I even added a handy dandy ESLint rule in the `.eslintrc` that will force you do to this!